app/assets/stylesheets/scaffolds.scss

```
@@ -93,6 +93,11 @@ body.ten-x-hacker-theme {
 
 body.trusted-status-true .trusted-visible-block {
   display: flex !important;
+  &.hidden {
+    @media screen and (max-width: $breakpoint-s) {
+      display: none !important;
+    }
+  }
```

This feels a bit hacky to me (using and overwriting importants).

I guess you need that `!important` in the first place because it needs to overwrite the other value that has `!important` flag. I think it could work if you removed that one important first:

```suggestion
body.trusted-status-true .trusted-visible-block {
  display: flex;
}
```

...and then, for the other change:

```
<span class="trusted-visible-block">
    <a class="crayons-header__link crayons-btn crayons-btn--ghost crayons-btn--icon-rounded m:flex hidden" ...>...</a>
</span>
```

^ it basically takes this `trusted-visible-block` logic out of the link itself and adds it to the wrapper (`span`) thanks to which we don't need to have `!important` conflicts and therefore we can simply drop importants altogether.

---

So adding the `span` seemed to work, and I had to keep the original styles of `display: flex !important;`:

```scss
body.trusted-status-true .trusted-visible-block {
  display: flex !important;
}
```

I'd rather not have `!important` too, but it seems to be the way to make this work without refactoring how we set the visibility of these mod links.